packages/wrangler/src/index.tsx

```diff
@@ -321,8 +324,64 @@ export async function main(argv: string[]): Promise<void> {
             console.log(
               `✨ Installed @cloudflare/workers-types.\nPlease add "@cloudflare/workers-types" to compilerOptions.types in your tsconfig.json`
             );
-          } else {
-            return;
+          }
+        }
+      }
+
+      if (isTypescriptProject) {
+        if (!fs.existsSync(path.join(process.cwd(), "src/index.ts"))) {
+          const shouldCreateSource = await confirm(
+            `Would you like to create a worker at src/index.ts?`
+          );
+          if (shouldCreateSource) {
+            const targetDir = path.join(process.cwd(), "src");
+            await mkdir(targetDir, { recursive: true });
+            await writeFile(
+              path.join(targetDir, "index.ts"),
+              `/** Welcome to Cloudflare Workers! This is your first worker.
+ *
+ * - Run \`wrangler dev src/index.ts\` in your terminal to start a development server
+ * - Open a browser tab at http://localhost:8787/ to see your worker in action
+ * - Run \`wrangler publish src/index.ts --name my-worker\` to deploy your worker
+ *
+ * Learn more at https://developers.cloudflare.com/workers/
+ */
+export default {
+  async fetch(request: Request) {
+    return new Response("Hello World!");
+  },
+};
```

Comment:
Ideally this should be just be a file that we can run checks on as part of our build/test run.
Then this code should just copy the file from this package to the users project.

Reply:
I'll move it, but we actually have to disable lint/type checks on them heh.